src/components/Organization/Organization.tsx

```diff
@@ -55,6 +58,59 @@ const Organization: React.FunctionComponent<Props> = ({
   const title = organization.name
     ? 'DataCite Commons: ' + organization.name
     : 'DataCite Commons: No Name'
+  
+  const statSummary = () => {
+    const works = pluralize(organization.works.totalCount, 'Work').split(' ')
+    const citations = pluralize(organization.citationCount, 'Citation').split(' ')
+    const views = pluralize(organization.viewCount, 'View').split(' ')
+    const downloads = pluralize(organization.downloadCount, 'Download').split(' ')
+
+    const stat = (count: string, name: string, link?: string) => {
+      if (Number(count) <= 0) return null
+
+      const helpIcon = () =>
+        // <OverlayTrigger 
+        //   placement="top"
+        //   overlay={<Tooltip id="tooltipAuthors">This list includes only Authors with ORCID ids.</Tooltip>}>
+          <a
+            href={`https://support.datacite.org/docs/${link}`}
+            target="_blank"
+            rel="noreferrer"
+            className='help-icon'
+          >
+            <FontAwesomeIcon icon={faQuestionCircle} />
+          </a>
+        // </OverlayTrigger>
+
+      return (
+        <Col xs={3} className="org-metadata">
+            <p className="count">{count}</p>
```

Comment:
I see this is the plain numeric count.  Sometimes we use a compact number (instead of 2500 it would be 2.5k)
For example in the [RepositoryDetail component](https://github.com/datacite/akita/blob/6be6e842f5c1a5dae39c0c84e0a89595260e161c/src/components/RepositoryDetail/RepositoryDetail.tsx#L377)

I double-checked the spec and it does not specify.  So I will approve but I want us to keep this in mind.

Reply:
It's an easy enough change to make it compact if needed. Also, on the wireframe it has the full number written out, but we can double check with Kristian when he gets back